ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py

```diff
@@ -160,6 +162,10 @@ def __init__(
         pipeline_name,
         incremental_configuration: IncrementalConfig,
     ):
+        self.default_entities = {
+            "database": {"excludes": DEFAULT_EXCLUDE_DATABASES},
+            "schema": {"excludes": DEFAULT_EXCLUDE_SCHEMAS},
+        }
```

Comment:
when we apply a default filter it will always be a exclude filter so i would prefer making it this way 


```suggestion
        self.default_exclude_filters = {
            "database": DEFAULT_EXCLUDE_DATABASES,
            "schema": DEFAULT_EXCLUDE_SCHEMAS,
        }
```

also instead of maintaining a dictionary just create a pydantic object for this purpose 

which we can initialise this common_db init and override in child classes if required